### DISCUSSION TITLE
数据隐私问题

### DISCUSSION THREAD

**Discussion by deict** [milvus-io/milvus#36320] [Q&A and General discussion]
如果我在windows本地部署milvus，我存储的这些数据会被你们官方接收存储吗

**Comment by yhmo** [comment]
不会，milvus完全开源，所有人都能看到代码，不存在这种行为.
milvus可以使用不同的存储系统，比如minio, gcp, aws, azure等等，你部署的milvus由你自己指定存储服务和存储位置，milvus只会往你指定的存储位置写数据。
